core/auth/src/main/java/software/amazon/awssdk/auth/signer/internal/AbstractAws4Signer.java

```diff
@@ -68,7 +68,7 @@ public abstract class AbstractAws4Signer<T extends Aws4SignerParams, U extends A
     private static final FifoCache<SignerKey> SIGNER_CACHE =
         new FifoCache<>(SIGNER_CACHE_MAX_SIZE);
     private static final List<String> LIST_OF_HEADERS_TO_IGNORE_IN_LOWER_CASE =
-        Arrays.asList("connection", "x-amzn-trace-id", "user-agent", "expect");
+        Arrays.asList("connection", "x-amzn-trace-id", "user-agent", "expect", "transfer-encoding");
```

Comment:
We have the same list in HeaderTransformsHelper used by AwsS3V4ChunkSigner, do we need to add TE there also?

Reply:
Nope , it is not required here because SDK add only one Header(singletonMap) via this function , that is checksum header and its always "x-amz-checksum-XYZ" 
https://github.com/aws/aws-sdk-java-v2/blob/6c74e87c0ef16472b33fb2dc33de6a18806dbe3a/core/auth/src/main/java/software/amazon/awssdk/auth/signer/internal/chunkedencoding/AwsS3V4ChunkSigner.java#L104-L106

Reply:
Done.